languages/python/oso/polar/host.py

```diff
@@ -17,6 +17,62 @@
 from .expression import Expression, Pattern
 
 
+# Entity classes
+
+
+class OsoResource:
```

Comment:
How about just `Resource` and `Actor` as the names?

Reply:
I suppose that's more likely to collide with an actual class... maybe? Or we namespace them in polar as `oso::Actor`. But that's basically the same thing.